presentation-pt-br.Rmd

```diff
@@ -0,0 +1,262 @@
+---
+title: "<img src= 'img/placeholder.png' width = '40%'></br></br>satRday"
+subtitle: "CIDADE"
+author: "DATA"
+#date: "12/12/2016"
+output:
+  xaringan::moon_reader:
+    lib_dir: libs
+    css: ["custom.css", "slides.css"]
+---
+
+```{r setup, include=FALSE}
+options(htmltools.dir.version = FALSE)
+```
+
+class: slide 
+
+## Sobre o satRday
+
+.pull-left[
+
+</br>
+</br>
+</br>
+<img src ="img/placeholder.png" width = "90%"></img>
+
+]
+
+.pull-right[
+
++ satRday é um __evento internacional__ que foi lançado em 2016.
+
++ O satRday foi patrocinado pela __RConsortium__, um projeto de código aberto da Linux Foundation que dá suporte à comunidade internacional de R.
+
++ É um projeto (relativamente) recente: nosso objetivo é organizar um evento por continente por trimestre.
+
++ Cada evento geralmente atrai entre 100 e 250 participantes.
+
++ Os eventos são sempre organizados com o envolvimento da comunidade local.
+]
+
+---
+class: slide 
+
+## A filosofia satRday
+#### Aberto, e Código Aberto
+
+Qualquer um pode organizar um satRday, desde que siga estes requisitos: 
+
++ O evento principal deve acontecer em um __Sábado__
+
++ O evento deve operar sob um __Código de Conduta__ e ter um __forte compromisso em aumentar a diversidade de palestrantes e participantes__
+
++ O preço do ingresso para participar do satRday de uma pessoa do setor industrial deve __custar menos de 8 horas de salário mínimo__ 
+
++ O evento tem uma __quantidade de ingressos gratuitos disponíveis para pessoas que não podem pagar__ o custo total
+
++ As pessoas palestrantes não devem ser obrigadas a pagar a taxa de inscrição
+
++ Deve haver um __mecanismo para incentivar e orientar novos(as) palestrantes__ 
+
++ O evento deve ser __organizado de forma voluntária__
+
++ __Os recursos em excesso devem ser utilizados para apoiar a comunidade local__ (grupos de usuários, RLadies, próximos satRdays, workshops, etc.) 
```

Comment:
Is not exactly as it was in the English version, but if it is translated completely, won't fit in the slides.